frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -96,9 +102,10 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
 
     const [volumeMode, setVolumeMode] = React.useState('');
     const onlyPvcSCs = (scObj: StorageClassResourceKind) =>
-      !snapshotClassResourceLoadError
-        ? scObj.provisioner.includes(snapshotClassResource?.driver)
-        : true;
+      !snapshotClassResourceLoadError &&
+      !scResourceLoadError &&
+      scObj.provisioner.includes(snapshotClassResource?.driver) &&
+      scObj.parameters?.encrypted === scResource?.parameters?.encrypted;
```

Comment:
```suggestion
      !scResourceLoadError ?
      scObj.provisioner.includes(scResource?.provisioner) &&
      scObj.parameters?.encrypted === scResource?.parameters?.encrypted
      : true;
```

Reply:
This returns true when there is a scResourceLoadError, and this may cause to add the sc to the dropdown. I hope it should be 'false' right ?